ReactAndroid/src/main/java/com/facebook/react/uimanager/events/ContentSizeChangeEvent.java

```diff
@@ -16,12 +18,26 @@
   private final int mWidth;
   private final int mHeight;
 
+  /**
+   * See {@link Event#Event(int)}.
+   *
+   * @param viewTag
+   * @param width
+   * @param height
+   */
+  @Deprecated
```

Comment:
> I don't think this scales well.

I agree.  I was originally thinking I could make this work with a deprecated constructor in `Event.java`; however, I realized that all of these events are exposed publicly.

> Do you have an idea how we could make events and tags work without patching any of them?

Not really.  I struggled with this because I wanted to avoid the duplication.  Any plugin that creates a `ContentSizeChangeEvent` with the view tag would break without this constructor. 

I think the `@Deprecated` constructors are needed.  Perhaps they ship with a few releases, then get removed with the hope that plugin maintainers have the chance to update their code?